cirq-core/cirq/circuits/moment.py

```diff
@@ -551,7 +551,7 @@ def cleanup_key(key: Any) -> Any:
         diagram.force_vertical_padding_after(1, 0)
 
         # Add operations.
-        for op in self.operations:
+        for op in sorted(self.operations, key=lambda op: op.qubits):
```

Comment:
Only thought is that we could cache this sorted object as it is used in equality as well.  But I'm fine if we don't do that.

Reply:
done, ready for another review / approval.